sentry/src/test/java/io/sentry/SentryTracerTest.kt

```diff
@@ -500,6 +501,29 @@ class SentryTracerTest {
         }
     }
 
+    @Test
+    fun `returns trace state without userId if not send pii`() {
```

Comment:
`l`: For full conditional coverage you could also add a test if the user is null and `isSendDefaultPii = true`, but maybe that's overkill. Up to you.

Reply:
added `returns baggage header without userId if send pii and null user` below